packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php

```diff
@@ -18,35 +19,31 @@ public function testSubRequest(): void
         $annotationReader = new AnnotationReader();
         $tokenManagerMock = $this->createMock(CsrfTokenManager::class);
 
-        $eventMock = $this->getMockBuilder(ControllerEvent::class)
-            ->disableOriginalConstructor()
-            ->setMethods(['isMasterRequest', 'getController'])
-            ->getMock();
-        $eventMock->expects($this->atLeastOnce())->method('isMasterRequest')->willReturn(false);
-        $eventMock->expects($this->never())->method('getController');
+        $event = new ControllerEvent(
+            $this->createMock(HttpKernelInterface::class),
+            new ControllerProtected(),
+            new Request(),
+            HttpKernelInterface::SUB_REQUEST,
+        );
 
         $routeCsrfProtector = new RouteCsrfProtector($annotationReader, $tokenManagerMock);
-        $routeCsrfProtector->onKernelController($eventMock);
+        $routeCsrfProtector->onKernelController($event);
```

Comment:
there are no assertions in this test now :thinking: I was surprised there is nothing like `assertExceptionIsNotThrown` in PHPUnit :smile: I discussed this with @grossmannmartin and we agreed that we would vote for adding `assertTrue(true)` at the end of the test, even thought we are not happy about it :smile:

Reply:
Fixed here: https://github.com/shopsys/shopsys/pull/2490/commits/2e480f6a69cef675227ae735ee86e90ae4461a27